app/views/oauth/authorized_applications/index.html.haml

```diff
@@ -7,7 +7,7 @@
 
 .applications-list
   - @applications.each do |application|
-    .applications-list__item
+    .applications-list__item{id: application.id}
```

Comment:
```suggestion
    .applications-list__item{ id: dom_id(application) }
```

Reply:
ahh good, I searched the whole codebase but couldn't find anywhere an anchor was was being set.